Source/JNA/pom.xml

```diff
@@ -202,6 +202,8 @@
                     <configuration>
                         <!-- Slightly faster builds, see https://issues.apache.org/jira/browse/MCOMPILER-209 -->
                         <useIncrementalCompilation>false</useIncrementalCompilation>
+                        <source>1.8</source>
```

Comment:
Thanks. I assume this is `signature.artifact` is Animal Sniffer? Upgraded it as well.

```
<signature.artifact>java18</signature.artifact>
```